modules/source/kafka/config/kafka.xml

```diff
@@ -68,23 +58,42 @@
 	</bean>
 
 	<bean id="kafkaInboundChannelAdapter"
```

Comment:
I'd rather make that a later enhancement (seeing that all this is internal) and keep current changes to a minimum,